src/server/coprocessor/endpoint.rs

```diff
@@ -802,10 +651,10 @@ impl SelectContextCore {
         }
 
         self.topn_heap.as_mut().unwrap().try_add_row(h,
-                                                     self.order_cols.clone(),
-                                                     self.ctx.clone(),
                                                      values,
-                                                     sort_keys)
+                                                     sort_keys,
+                                                     self.order_cols.clone(),
+                                                     self.ctx.clone())
```

Comment:
try_add_row is only used in topn executor, and the code in the endpoint will be refactored or removeed later by being splited into serval executor.